openc3/lib/openc3/models/cvt_model.rb

```diff
@@ -93,7 +93,7 @@ def self.get_item(target_name, packet_name, item_name, type:, scope:)
     # @param stale_time [Integer] Time in seconds from Time.now that value will be marked stale
     # @return [Array] Array of values
     def self.get_tlm_values(items, stale_time: 30, scope: $openc3_scope)
-      now = Time.now.to_f
+      now = Time.now.sys.to_f
```

Comment:
This felt like a bug but now I'm not sure. We're comparing it to RECEIVED_TIME. I think we need more testing with UTC vs local time.